packages/vincent-tools/vincent-tool-sdk/src/lib/types.ts

```diff
@@ -243,3 +244,31 @@ export interface VincentToolDef<
     policyEvaluationResults: VincentPolicyEvaluationResults<Policies>,
   ) => Promise<unknown>;
 }
+
+export interface PolicyParameter {
```

Comment:
Since package consumers should only get the post-decoding and parsing values, we need to make sure that these aren't exported from the package. 🤔 Maybe we can move them into a `types.ts` file inside of the `Vincent-policy`  instead of it being in our root types.ts

Reply:
Moved them to `lit-actions/types.ts`